app/views/controllers/shared/_form_list_feedback.erb

```diff
@@ -48,7 +48,7 @@ if !@multiple_names.blank?
       @multiple_names.each do |name|
         concat(tag.div(name.display_name.t))
         concat(fields_for(:chosen_multiple_names) do |f_c|
-          name.other_authors.each do |other_name|
+          name.other_authors.includes([:observations]).each do |other_name|
```

Comment:
This looks like a db query, which i think we should be aiming to not do at the view layer. 

Can we eager load this association in `SharedPrivateMethods` as suggested above, and do here:
```ruby
@multiple_names.each do |name, other_authors|
  concat(tag.div(name.display_name.t))
  concat(fields_for(:chosen_multiple_names) do |f_c|
    other_authors.each do |other_name|
    # ...
```

Reply:
Done.  Not clear to me that it's worth the extra code in the controller, but whatever.